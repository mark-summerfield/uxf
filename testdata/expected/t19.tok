FileComment Str("Suitable for SQLite conversion")
TClassBegin ttype=<PriceList>
  Field Str("Date")
  Field Str("Price")
  Field Str("Quantity")
  Field Str("ID")
  Field Str("Description")
TClassEnd
TableBegin ttype=<PriceList>
  Date(2022-09-21)
  Real(3.99)
  Int(2)
  Str("CH1-A2")
  Str("Chisels (pair), 1in & 1Â¼in")
  Date(2022-10-02)
  Real(4.49)
  Int(1)
  Str("HV2-K9")
  Str("Hammer, 2lb")
  Date(2022-10-02)
  Real(10.0)
  Int(1)
  Str("SX4-D1")
  Str("Eversure Sealant, 13-floz")
TableEnd
Eof
----------------------------------------
