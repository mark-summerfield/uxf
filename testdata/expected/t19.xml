<?xml version="1.0" ?>
<uxf version="1.0">
  <comment>Suitable for SQLite conversion</comment>
  <ttypes>
    <ttype name="PriceList">
      <field name="Date"/>
      <field name="Price"/>
      <field name="Quantity"/>
      <field name="ID"/>
      <field name="Description"/>
    </ttype>
  </ttypes>
  <table name="PriceList">
    <row>
      <date v="2022-09-21"/>
      <real v="3.99"/>
      <int v="2"/>
      <str>CH1-A2</str>
      <str>Chisels (pair), 1in &amp; 1Â¼in</str>
    </row>
    <row>
      <date v="2022-10-02"/>
      <real v="4.49"/>
      <int v="1"/>
      <str>HV2-K9</str>
      <str>Hammer, 2lb</str>
    </row>
    <row>
      <date v="2022-10-02"/>
      <real v="10.0"/>
      <int v="1"/>
      <str>SX4-D1</str>
      <str>Eversure Sealant, 13-floz</str>
    </row>
  </table>
</uxf>
